### DISCUSSION TITLE
Setting pimcore checkbox to checked as default

### DISCUSSION THREAD

**Discussion by nemanjavx** [pimcore/pimcore#10861] [Q&A / Support] [answered]
Is it possible to render the pimcore_checkbox() as checked by default?  have tried to find a solution for this, but there seems to be no option for it. Nothing is also mentioned in the docs. So, is something like this actually possible, and if so, can you provide some kind of information on how to do it and where to look?

**Comment by dvesh3** [comment, marked as answer]
@NemanjaV99 Sorry, there is no default value option for checkbox editable. Still you can try something like `{% do pimcore_checkbox('myCheckBox').setDataFromResource(1) %}`

**Reply by nemanjavx** [reply]
Alright, thank you very much for the answer! @dvesh3

**Reply by marcohefti** [reply]
Would be amazing to have a way to set the value per default cause I stumble across this problem every now and then. Basically the only solution right now is to build a new feature for an existing component around having false per default
